index.js

```diff
@@ -46,8 +46,8 @@ cs.get.rgb = function (string) {
 		return null;
 	}
 
-	var abbr = /^#([a-fA-F0-9]{3})$/;
-	var hex = /^#([a-fA-F0-9]{6})$/;
+	var abbr = /^#([a-fA-F0-9]{3,4})$/;
+	var hex = /^#([a-fA-F0-9]{6,8})$/;
```

Comment:
Could you adjust this to `/^#([a-f0-9]{6})([a-f0-9]{2})?$/i`? The logic below would have to be updated I believe as well.